library/src/lib/shellbar/shellbar-actions/shellbar-actions.component.scss

```diff
@@ -0,0 +1,4 @@
+.fd-search-input--closed .fd-popover__body {
+    width: 300px !important;
+    left: -264px !important;
```

Comment:
These styles are normally applied to this class but are overridden by popper.js

Reply:
This is very specific and not very responsive

Reply:
This is exactly how the component is styled in Fundamental, minus the !important, which is necessary because popper.js is overriding these styles.  I think shellbar responsiveness is a larger separate issue